build-system/tasks/bundle-size.js

```diff
@@ -97,7 +119,7 @@ function storeBundleSize(bundleSize) {
  * Checks gzipped size of existing v0.js (amp.js) against `maxSize`.
  * Does _not_ rebuild: run `gulp dist --fortesting --noextensions` first.
  */
-function checkBundleSize() {
+async function checkBundleSize() {
```

Comment:
Love seeing more `async` methods here, makes this quite easy to read. Nice work.